docs/attributes-dictionary/README.md

```diff
@@ -0,0 +1,27 @@
+<!--- Hugo front matter used to generate the website version of this page:
+linkTitle: Registry
+weight: -2
+--->
+
+# Attributes Registry
+
+The attributes registry is the place where attributes are defined. An attribute definition covers the following properties of an attribute:
+
+- the fully qualified name (i.e. `id`) of the attribute
```

Comment:
The `id` is a bit confusing here, as I interpreted it as an example for a "fully qualified name", which I think it's not, but I guess it rather is referring to the YAML keys.

I wonder if something like this can be clearer:
```suggestion
- the `id` (the fully qualified name) of the attribute
```